### ISSUE TITLE
Gorename returns error, "Sorry, rename failed to execute"

### ISSUE THREAD

**Issue by hjma29** [Microsoft/vscode-go#1087]
code: 1.14.0
vscode-go: 0.6.62

Sometimes I get the above error. So far I feel if the symbol starts with Capital letter, it will fail. I can't find more logs in VS code for this error.

**Comment by markphelps** [comment]
Same issue:
* Code: Version 1.14.1
* VS-Go: 0.6.62
* go1.8.3 darwin/amd64

Here's a [log](https://gist.githubusercontent.com/markphelps/5e7fbcecdab9170e43d4a4ab7c53b556/raw/7dee73018a638b5323dd467d9ff028a0aacd930d/go-rename-failed.log) I got from `Help -> Toggle Developer Tools -> Console`

I got the same error when running: `$ gorename -from '"github.com/markphelps/optional".OptionalString' -to String` on the command line so I don't think it's an issue with VS-Code or VS-Code-Go.

After doing some googling I came across [this post](https://groups.google.com/forum/#!topic/golang-nuts/fqBpBqtUiFQ) on the golang-nuts email list.

**tl;dr**: Run `$ go get -u golang.org/x/tools/cmd/gorename` to update gorename to the latest version or re-install

After I did that it worked on the command line and in VS-Code

**Comment by hjma29** [comment]
Thank you very much. After knowing where to look for log from your post, I can see gorename scanned my other testing directories(Because my symbol has capital letter) under GOPATH and found some errors under those directories(code was not cleanedup yet). I deleted those files under testing directory and now rename works.

**Comment by ramya-rao-a** [comment]
Thanks for the update @markphelps and @hjma29 
Glad both of your issues go resolved.

Happy Coding!
